earthquake-engineering-and-structural-dynamics.csl

```diff
@@ -148,8 +148,8 @@
                 <text variable="container-title" text-case="title" strip-periods="true" font-style="italic"/>
               </group>
               <group>
-                <text macro="year-date" suffix="; "/>
-                <text variable="volume" font-weight="bold"/>
+                <text macro="year-date"/>
+                <text variable="volume" font-weight="bold" prefix="; "/>
                 <text variable="issue" prefix="(" suffix=")"/>
```

Comment:
I think this will actually print the semicolon in bold, which, while hard to see, isn't right. Could you use

```
<group delimiter="; ">
   <text macro="year-date"/>
   <text variable="volume" font-weight="bold"/>
</group>
```

instead?